CONTRIBUTING.md

```diff
@@ -41,15 +41,16 @@ Creating a pull request is fairly simple, just make sure it focuses on a single
 
 We would greatly appreciate the code submitted to be of a consistent style with other code in disnake. This project follows PEP-8 guidelines (mostly) with a column limit of 100 characters.
 
-We use [`nox`](https://nox.thea.codes/en/stable/) for automating development tasks. Run these commands to
-install `nox` and `taskipy` as well as the required dependencies in your environment,
-and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.  
-Make sure you have a virtualenv activated if you don't want it to install the packages globally!
+
+
+Run these commands to install `nox`, `pdm`, and `taskipy` as well as the required dependencies in your environment, and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.  
```

Comment:
Would be great to have a short section on how to manage (add/remove/update) packages with pdm, and add hyperlinks to the docs of nox/pdm/taskipy.